lib/FieldType/DataTransformer/SelectionValueTransformer.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\FieldType\DataTransformer;
+
+use eZ\Publish\Core\FieldType\Selection\Value;
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * DataTransformer for Selection\Value.
+ * Needed to display the form field correctly and transform it back to an appropriate value object,
+ * as single selection form fields return integer values, while multiselects return arrays.
+ */
+class SelectionValueTransformer implements DataTransformerInterface
```

Comment:
Then it is pretty much the same reason: make it testable ;)

Go for it if you think it is useful.